File: tests/talos/java/orderbook/src/main/java/com/talostrading/OrderBook.java
package com.talostrading;

import java.util.List;

/**
 * OrderBook maintains a set of orders by price level.
 */
public class OrderBook implements IOrderBook {

    @Override
    public void onMessage(final OrderUpdate orderUpdate) {
        throw new java.lang.UnsupportedOperationException("Not implemented yet.");
    }

    @Override
    public List<PriceLevel> bids() {
        throw new java.lang.UnsupportedOperationException("Not implemented yet.");
    }

    @Override
    public List<PriceLevel> offers() {
        throw new java.lang.UnsupportedOperationException("Not implemented yet.");
    }
}

File: tests/talos/golang/orderbook/orderbook.go
package orderbook

type OrderBook struct {
}

func NewOrderBook() *OrderBook {
	return &OrderBook{}
}

func (b *OrderBook) OnMessage(msg OrderUpdate) {
	return
}

func (b *OrderBook) Bids() []PriceLevel {
	return nil
}

func (b *OrderBook) Offers() []PriceLevel {
	return nil
}
